core/src/test/resources/application.conf

```diff
@@ -11,3 +11,5 @@ database = {
 messaging {
   mode = "test"
 }
+
+packages.versionFormat = ".+"
```

Comment:
The reason I added this was not because I got any weird errors, it's that the default package version format is a semantic version, which is compatible with treehub packages.